### DISCUSSION TITLE
Error: Uncaught ReferenceError: __webpack_require__ is not defined at

### DISCUSSION THREAD

**Discussion by itayeylath** [webpack/webpack#18135] [Q&A]
My OS is Linux-mint and I have a React typescript app. working with yarn (not npm) and docker containers for APISIX and front-end. Also, I have Webpack but without out Webpack configuration file.

I researched solutions on the web and did not find something that worked for me. I also restarted the computer, removed and restarted for docker, and removed and cloned the repository again.

Can you give me solutions with instructions and examples of what to do? :)

After executing 'yarn dev', the server is up, and I get the localhost port. when I open the localhost URL on Chrom the UI is white, and I do not see the app.

In the console getting the error ''Uncaught ReferenceError: webpack_require is not defined at u/newrelic"

**Comment by AdnanMehr8** [comment]
It seems like you're encountering an issue with your React TypeScript app when running it locally using Yarn and Docker containers. The error you mentioned, "Uncaught ReferenceError: webpack_require is not defined at u/newrelic", suggests that there might be an issue with the bundling or configuration of your webpack setup.

**Comment by itayeylath** [comment]
> It seems like you're encountering an issue with your React TypeScript app when running it locally using Yarn and Docker containers. The error you mentioned, "Uncaught ReferenceError: webpack_require is not defined at u/newrelic", suggests that there might be an issue with the bundling or configuration of your webpack setup.

Thanks! do you have suggestions on how to debug the bundling or configuration?

**Comment by alexander-akait** [comment]
Looks like you forgot to include runtime chunk, without configuration file or additional infromation we can't help, sorry, if you can provide a reproducible test repo I will help

**Reply by itayeylath** [reply]
this is my _webpack.config.js_ inside _node_modules_.

```
'use strict';

const path = require('path');

module.exports = {
    entry: path.resolve(__dirname, 'src/index.js'),
    output: {
        path: path.resolve(__dirname, 'assets'),
        filename: 'bundle.js'
    },
    module: {
        rules: [
            {
                test: /\.js$/,
                exclude: /node_modules/,
                use: {
                    loader: 'babel-loader'
                }
            }
        ]
    }
};
```

**Reply by alexander-akait** [reply]
Can you show me the dist directory and your HTML file/template/etc

**Reply by itayeylath** [reply]
thank you but I fixed it by changing it from-
` "@newrelic/browser-agent": "^1.240.0"`
to-
` "@newrelic/browser-agent": "1.240.0"`
 
 without the ^
